v2/pkg/variablesvalidation/variablesvalidation.go

```diff
@@ -11,8 +11,13 @@ import (
 	"github.com/wundergraph/graphql-go-tools/v2/pkg/operationreport"
 )
 
+const (
+	BadUserInput = "BAD_USER_INPUT"
```

Comment:
proposal

should we create separate package `errorcodes` with const.go? to use it like this

```
errorcodes.GraphQLValidationFailed
errorcodes.BadUserInput
```

Reply:
with this approach we don't need ErrorCode prefix to const name

Reply:
we could also move there `InvalidGraphqlErrorCode` from resolve package